AnnoyancesFilter/sections/subscriptions.txt

```diff
@@ -33,7 +33,7 @@ turkuazgazetesi.net,yenialanya.com,mansetaydin.com,seskocaeli.com,kocaelibarisga
 ||embeds.fanmatics.com^$third-party
 ||cp.unisender.com/*/form-widget-loader.js$third-party
 ##.widget_blog_subscription
-~nothing.tech,~mega-debrid.eu,~curvegames.com,~news.un.org,~norwichzen.org.uk,~dereferer.me###mc_embed_signup
+~copays.org,~nothing.tech,~mega-debrid.eu,~curvegames.com,~news.un.org,~norwichzen.org.uk,~dereferer.me###mc_embed_signup
```

Comment:
What's the reason to unblock it? It's still a subscription. User can disable AdGuard Annoyances.

Reply:
I see that this subscription is only here https://copays.org/stay-connected/, so it's ok to unblock it.